src/Map/Map.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <map>
+
+using namespace std;
+
+/************************************************************ Continent ************************************************************/
+
+class Continent {
+private:
+    int id;
+    string name;
+
+public:
+
+    /// <summary>
+    /// Default Constructor
+    /// </summary>
```

Comment:
Can we get rid of these comments in the future? And just add them at end of the same line if necessary, instead of on top?

Reply:
<img width="876" alt="image" src="https://github.com/LaraMo/Risk-Swift-Sloths/assets/31352301/7f76d607-a1d9-4f61-affc-9b7b493bce6c">
after reading, this is also a good standard if you prefer to comment the methods like so, let me know:)